### ISSUE TITLE
以下代码会存在if first_frame_file is not None:结果被覆盖的情况，是不是写错了。

### ISSUE THREAD

**Issue by zfnice** [Kevin-thu/StoryMem#18]
if first_frame_file is not None:
            u = self.vae.encode(
                torch.concat([
                    torch.nn.functional.interpolate(first_frame[None].cpu(), size=(h, w), mode='bicubic').transpose(0, 1),
                    torch.zeros(3, F-1, h, w)
                ], dim=1).unsqueeze(0).to(self.device)
            ).float().squeeze(0)
        if motion_frames_file is not None:
            u = self.vae.encode(
                torch.concat([
                    motion_frames.transpose(0, 1).to(self.device),
                    torch.zeros(3, F-5, h, w).to(self.device)
                ], dim=1).unsqueeze(0)
            ).float().squeeze(0)
        else:
            u = torch.zeros(3, F, h, w).to(self.device)
            u = self.vae.encode(u.unsqueeze(0)).float().squeeze(0)

**Comment by FaJingyi** [comment]
不会啊，你看外面的逻辑，进到generate里后至少有一个会是None，或者两个全是None

**Comment by zfnice** [comment]
> 不会啊，你看外面的逻辑，进到generate里后至少有一个会是None，或者两个全是None

如果first_frame_file is not none,u计算了一次，接下来如果 motion_frames_file is  none, u 又会重新赋值，之前计算的就没用了。

**Comment by FaJingyi** [comment]
嗷嗷我懂你意思了，这边进来的时候虽然如果first_frame_file 先计算了一次u, motion_frames_file 是None,但是又会走else再算一次，第一次被覆盖了，确实有问题

**Comment by Kevin-thu** [comment]
Thanks for pointing this out! We have fixed it.
